src/Core/Resolvers/CosmosQueryBuilder.cs

```diff
@@ -162,7 +162,7 @@ private static string Build(Stack<CosmosJoinStructure> joinstructure)
                     continue;
                 }
 
-                joinBuilder.Append($" JOIN {structure.TableAlias} IN {structure.DbObject.SchemaName}.{structure.DbObject.Name}");
+                joinBuilder.Append($" JOIN {structure.TableAlias} IN {structure.DbObject.FullName}");
```

Comment:
Were there any tests failing afresh that we are required to fix this?

Reply:
No, tests were not failing due to this one, I found that `FullName` already has this information which I am constructing here.